pallets/liquidity-pools/src/lib.rs

```diff
@@ -951,6 +946,12 @@ pub mod pallet {
 			let domain_address = T::DomainAccountToDomainAddress::convert(domain_account);
 			T::DomainAddressToAccountId::convert(domain_address)
 		}
+
+		fn outbound_weight(reads: u64, writes: u64) -> Weight {
+			Weight::from_parts(1_000_000, 256) //defensive base weights
+				.saturating_add(T::DbWeight::get().reads_writes(reads, writes))
+				.saturating_add(T::OutboundQueue::weight())
+		}
```

Comment:
This takes into account the correct `OutboundQueue::submit()` weights even when the message is a batch. We no longer need `defensive_weights.rs`